### PR TITLE
Citra-Wiki-Optimization Re-Pulled with less commits

### PR DESCRIPTION
**Pull request by robzombie91**

### CODE REVIEW

**Comment by robzombie91** [comment]
At least it passed travis now

File: games/fire-emblem-warriors/game.dat [hunk 1]
```diff
@@ -13,6 +13,11 @@ title = "000400000F70CD00"
 region = "eur"
 release_date = "2017-10-20"
 
+[[releases]]
+title = "000400000F70C100"
+region = "jpn"
+release_date = "2017-08-28"
```
Comment (Leo626) [inline games/fire-emblem-warriors/game.dat]: Wrong date.

> JP: September 28, 2017

File: games/pokemon-rumble-world/game.dat [hunk 2]
```diff
@@ -1,13 +1,23 @@
-title = "Pokémon Rumble World"
-description = "Pokémon Rumble World is a freemium action RPG developed by Ambrella and published by The Pokémon Company. It is the fourth entry in the Pokémon Rumble series."
+title = "Pokemon Rumble World"
```
Comment (Leo626) [inline games/pokemon-rumble-world/game.dat]: Accent in the e is suppose to be there.

File: games/pokemon-rumble-world/game.dat [hunk 3]
```diff
@@ -1,13 +1,23 @@
...
+description = "Pokemon Rumble World is a freemium action RPG developed by Ambrella and published by The Pokemon Company. It is the fourth entry in the Pokemon Rumble series."
```
Comment (Leo626) [inline games/pokemon-rumble-world/game.dat]: Ditto

File: games/star-fox-64-3d/game.dat [hunk 4]
```diff
@@ -9,6 +9,16 @@ title = "0004000000049000"
...
 
+[[ releases ]]
+title = "0004000000049100"
+region = "eur"
+release_date = "2011-09-09"
+
+[[ releases ]]
+title = "0004000000030400"
+region = "jpn"
+release_date = "2011-06-14"
```
Comment (Leo626) [inline games/star-fox-64-3d/game.dat]: Wrong date.

> JP: July 14, 2011

File: games/story-of-seasons-trio-of-towns/game.dat [hunk 5]
```diff
@@ -9,6 +9,16 @@ title = "000400000019F500"
...
 
+[[ releases ]]
+title = "00040000001B5500"
+region = "eur"
+release_date = "2017-10-13"
+
+[[ releases ]]
+title = "0004000000185300"
+region = "jpn"
+release_date = "2016-05-23"
```
Comment (Leo626) [inline games/story-of-seasons-trio-of-towns/game.dat]: Wrong date.

> JP: June 23, 2016

File: games/tomodachi-life/game.dat [hunk 6]
```diff
@@ -4,11 +4,21 @@ github_issues = [2475]
 needs_system_files = true
 needs_shared_font = true
 
-[[releases]]
+[[ releases ]]
+title = "000400000008C300"
+region = "usa"
+release_date = "2014-05-06"
```
Comment (Leo626) [inline games/tomodachi-life/game.dat]: Wrong date.

> NA: June 6, 2014

**Comment by Leo626** [comment]
* Still need to add eShop release for Pokemon Rumble World.
* While reviewing I noticed wrong eur release date for Fire Emblem Awakening and usa release dates for Mario Sports Superstars and Star Fox 64 3D. Would be nice to correct those while you're at it.
* Do you plan to address Breadfish's comments regarding the boxarts. Most of them seem unnecessary. There isn't much noticeable differences between them. Granted they must be compressed as noted in the contributing guide. The change to USA boxarts and final ESRB ratings ones are fine.

**Comment by robzombie91** [comment]
Some of the box-arts had the outline of the box, just a minor annoyance but enough to warrant a different one. While I was initially changing them I accidentally set them at the wrong size, so I resized them to the contribution guides recommended size but that had the unintended consequence of making them look worse. I plan to go back through the rest of them later on since I just bough FF-XV for PC and intend to play it for as long as possible.

Going to push your fixes in right now.

**Comment by robzombie91** [comment]
Are there any other changes needed?

File: games/donkey-kong-country-returns-3d/game.dat [hunk 7]
```diff
@@ -1,5 +1,5 @@
 title = "Donkey Kong Country Returns 3D"
-description = "Donkey Kong Country Returns is a 2010 side-scrolling platformer videogame developed by Retro Studios and published by Nintendo for the Wii console. A stereoscopic port of the game, titled Donkey Kong Country Returns 3D, was released for the Nintendo 3DS in Europe and North America on May 24, 2013, in Australia on May 25, 2013, and in Japan on June 13, 2013."
+description = "Donkey Kong Country Returns 3D is a side-scrolling platformer developed by Monster Studios and published by Nintendo. This game was originally released on the Wii."
```
Comment (Leo626) [inline games/donkey-kong-country-returns-3d/game.dat]: The last sentence could be said better.
> It is a port of the Wii title Donkey Kong Country Returns, with some additional content

**Comment by Leo626** [comment]
@robzombie91 I plan to review this one more time later.

**Comment by robzombie91** [comment]
OK NP. I'll implement your changes once I get home from work.

**Comment by Leo626** [comment]
Fantasy Life has an enhanced Japan re-release called Fantasy Life Link!. It should be added.

File: games/paper-mario-sticker-star/game.dat [hunk 8]
```diff
@@ -9,16 +9,16 @@ title = "00040000000A5E00"
...
 
-[[ releases ]]
-title = "00040000000A5D00"
-region = "jpn"
-release_date = "2012-12-06"
-
 [[ releases ]]
 title = "00040000000A5F00"
 region = "eur"
 release_date = "2012-12-08"
```
Comment (Leo626) [inline games/paper-mario-sticker-star/game.dat]: This date is wrong. 

> EU: December 7, 2012

File: games/pokemon-rumble-world/game.dat [hunk 9]
```diff
@@ -1,12 +1,27 @@
-title = "Pokémon Rumble World"
-description = "Pokémon Rumble World is a freemium action RPG developed by Ambrella and published by The Pokémon Company. It is the fourth entry in the Pokémon Rumble series."
+title = "Pok�mon Rumble World"
```
Comment (Leo626) [inline games/pokemon-rumble-world/game.dat]: What is wrong with the accented e? Try copying and pasting it instead.

File: games/pokemon-rumble-world/game.dat [hunk 10]
```diff
@@ -1,12 +1,27 @@
...
+description = "Pok�mon Rumble World is a freemium action RPG developed by Ambrella and published by The Pok�mon Company. It is the fourth entry in the Pok�mon Rumble series."
```
Comment (Leo626) [inline games/pokemon-rumble-world/game.dat]: Ditto.

File: games/pokemon-super-mystery-dungeon/game.dat [hunk 11]
```diff
@@ -14,6 +14,11 @@ title = "0004000000174400"
 region = "eur"
 release_date = "2016-02-20"
```
Comment (Leo626) [inline games/pokemon-super-mystery-dungeon/game.dat]: This date is wrong.

> EU: February 19, 2016

File: games/pokemon-rumble-world/game.dat (hunk 9 above)
Comment (robzombie91) [reply games/pokemon-rumble-world/game.dat]: i tried to and thats what came out

**Comment by j-selby** [comment]
This is having the same issues as your previous PR. A good PR is one which limits its scope to one particular item. Generic fixes should be broken up and submitted individually. Further, commits should also be readable individually - "Travis Test" is perfectly fine for testing, but you should be squashing when you want to actually PR those changes.

**Comment by robzombie91** [comment]
The Travis test commits where because Travis was having issues with notepad for whatever reason so i had to recreate the files in notepad++ to get Travis to build correctly.

@j-selby

**Comment by j-selby** [comment]
```
Notepad++ test and Fixes 853pm are the true commits that implement what leo wanted with his last few comments.
```

My point was that these commits should be squashed down and named appropriately then. Perfectly fine to debug CI through throwaway commits, but when you want to present your stuff, not so much.

File: games/pokemon-rumble-world/game.dat [hunk 12]
```diff
@@ -5,8 +5,23 @@ needs_system_files = true
...
+title = "0004000000185A00"
+region = "eur"
+release_date = "2016-01-22"
+
+[[ releases ]]
+title = "0004000000185A00"
+region = "jpn"
+release_date = "2015-11-19"
+
+[[ releases ]]
```
Comment (CodingKoopa) [inline games/pokemon-rumble-world/game.dat]: What's with this `all` release?

Comment (Leo626) [reply games/pokemon-rumble-world/game.dat]: For some reason this game has two title IDs. One for retail and one eShop (which is what the `all` release is). The existing testcsae there has that title ID.

**Pull request merged**